pkg/nsx/cluster.go

```diff
@@ -265,9 +265,13 @@ func (nsxVersion *NsxVersion) Validate() error {
 func (nsxVersion *NsxVersion) featureSupported(feature string) bool {
 	var minVersion [3]int64
 	validFeature := false
-	if feature == FeatureSecurityPolicy {
+	switch feature {
+	case FeatureSecurityPolicy:
 		minVersion = nsx320Version
 		validFeature = true
+	case FeatureNSXServiceAccount:
+		minVersion = nsx410Version
```

Comment:
minVersion should be changed to NSX 4.0.1. The VPC based multi-tenancy is available in NSX 4.0.1 already.

Reply:
Fixed.